environment.yml

```diff
@@ -4,17 +4,18 @@ channels:
   - conda-forge
 dependencies:
   - astropy>=4.0
-  - h5py
+  - astropy-healpix
```

Comment:
So it's required if you want to use healpix catalogs or healpix beams. But it's an optional, not required dependency of pyradiosky and pyuvdata, so it has to be listed explicitly here.

Reply:
I also clarified this a bit better in the readme.

Reply:
this change is good for me. I'll leave for adam to put the big green check on.

Reply:
I think you're correct about this, but we don't have any split uvdata feedstocks on conda-forge to select optional dependencies. Otherwise we could install `pyuvdata-beams` or something similar and have that healpix-enabled uvdata be the real dependecy.

Reply:
a relevant conda issue https://github.com/conda/conda/issues/2984